src/ce.js

```diff
@@ -40,8 +40,9 @@ var ce = {
             'tag_name': elem.tagName
         };
 
-        if (_.includes(['input', 'select', 'textarea'], elem.tagName.toLowerCase())) {
-            props['value'] = this._getFormFieldValue(elem);
+        var formFieldValue = this._getFormFieldValue(elem);
```

Comment:
<img width="662" alt="screen shot 2016-08-08 at 5 03 29 pm" src="https://cloud.githubusercontent.com/assets/7527733/17500384/e23e4e76-5d89-11e6-8e2b-261800833c80.png">
because this